docs: fix linter errors for what-is-angular.md (#47708)
angular/angular: aio/content/guide/what-is-angular.md

<code-example format="html" language="html" path="what-is-angular/src/app/hello-world-example.html"></code-example>

Angular's component model offers strong encapsulation and an intuitive application structure.
Components also make your application painless to unit test and can improve the general readability of your code.

For more information on what to do with components, see the [Components](guide/component-overview) section.

<a id="templates"></a>

### Templates

<!-- vale Angular.Google_WordListWarnings = NO -->

Every component has an HTML template that declares how that component renders.
You define this template either inline or by file path.

<!-- vale Angular.Google_WordListWarnings = YES -->

> Angular extends HTML with extra syntax that lets you insert dynamic values from your component.

Reviewer: How about:
```suggestion
Angular adds syntax elements that extend HTML so you can insert dynamic values from your component .
```

    <code-pane header="hello-world-bindings.component.html" path="what-is-angular/src/app/hello-world-bindings/hello-world-bindings.component.html"></code-pane>
</code-tabs>

> Add extra features to your templates through the use of [directives](guide/built-in-directives).

Reviewer: If you're adding them, they're additional, and extra. :smile:
```suggestion
Add features to your templates by using [directives](guide/built-in-directives).
```

The Angular CLI makes some tasks trouble-free.
For example:

<!-- vale Angular.Google_WordListSuggestions = NO -->

| Command                     | Details |
|:---                         |:---     |
| [ng build](cli/build)       | Compiles an Angular application into an output directory.                     |
| [ng serve](cli/serve)       | Builds and serves your application, rebuilding on file changes.       |
| [ng generate](cli/generate) | Generates or modifies files based on a schematic.                     |
| [ng test](cli/test)         | Runs unit tests on a given project.                                   |
| [ng e2e](cli/e2e)           | Builds and serves an Angular application, then runs end-to-end tests. |

<!-- vale Angular.Google_WordListSuggestions = YES -->

The Angular CLI is a valuable tool for building out your applications.

For more information about the Angular CLI, see the [Angular CLI Reference](cli) section.

<a id="1p-libraries"></a>

## First-party libraries

The section, [Angular applications: the essentials](#essentials), provides a brief overview of a couple of the key architectural elements that are used when building Angular applications.

> The many benefits of Angular really become clear when your application grows and you want to add more functions such as site navigation or user input.

Reviewer: How about:
```suggestion
The many benefits of Angular really become clear when your application grows and you want to add functions such as site navigation or user input.
```
`more` (or `additional`) isn't necessary.
